bouncer/util.py

```diff
@@ -44,15 +45,31 @@ def parse_document(document):
     annotation_id = document["_id"]
     annotation = document["_source"]
 
+    # And that annotations always have "group" and "shared"
+    group = annotation["group"]
+    shared = annotation["shared"]
```

Comment:
Since we're dealing with potentially exposing private information maybe we can be a bit more defensive here and change the code to `shared = annotation['shared'] == True`.

If, for whatever reason, we would change the value of `shared` in the search index to a string, then we would expose private data because a non-empty string is always truthy.